src/Administration/Resources/app/administration/fix_components_export.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
that file is a script to refactor components globally, so remove sideefects (more about that below)